public/js/controllers/game.js

```diff
@@ -320,21 +330,17 @@ angular //eslint-disable-line
           }
         }
 
-
         $scope.sendChatMessage = function (message) {
           if (message) {
             $scope.chats.$add({
               image: game.players[game.playerIndex].avatar,
               message: $scope.message,
               date: Date.now(),
               user: game.players[game.playerIndex].username
+            }).then(() => {
```

Comment:
I can see that the .then() block has no statements in it. I suggest you remove it since it does nothing.

Reply:
Ok, thanks for pointing it out